downstairs/src/lib.rs

```diff
@@ -1727,23 +1686,39 @@ impl DownstairsBuilder<'_> {
             region.def().extent_count(),
         );
 
+        let (request_tx, request_rx) = mpsc::channel(256); // TODO size?
```

Comment:
Probably need to revisit this, unless this channel goes away in a future commit?

Reply:
Good catch; I switch to an `unbounded_channel` later on in this stack of commits (because it's eventually limited by Upstairs backpressure)